### ISSUE TITLE
Landing on Compare tab when navigating to new entity page

### ISSUE THREAD

**Issue by mbrush** [monarch-initiative/monarch-legacy#920]
Recreate: From phenotype tab on disease page (http://beta.monarchinitiative.org/disease/DOID:863) click on a disease to go to its page.  You land on the Compare tab (I would expect to land on the Overview tab).  And the tabs are out of order.  Compare is on far left and Overview is on far right. 

This seems to happen whenever navigating from one entity to another in this way (e.g. navigating from a disease to a gene, or variant).   My expected behavior is that I would always land on the Overview tab, and the tabs would always be in same order.

**Comment by kshefchek** [comment]
This is intentional, but repositioning the tabs is trivial.  But for now this is what is coded:
- All pages except phenotype: Compare tab on far left, overview on far right
- If a page has no phenotypes, or a disease page with over 10k phenotypes: Land on overview tab, compare tab greyed out
- Disease page where a disease is a grouping class or phenotype page - Overview on far left, compare on far right

Basically, we always land on the left most tab, but we could put the compare/overview tab next to it instead.

**Comment by mellybelly** [comment]
I was indeed thinking that on some page I was on it was odd that the omim description of the disease was way over on the right most tab, whilst the compare tab was first. probably some of this content should eventually go in the jumbotron and the "overview" tab could become an ontology navigation tab.

**Comment by kshefchek** [comment]
Would it make more sense to put the overview tab next to the compare tab?

**Comment by mbrush** [comment]
Ok.  For ordering, can we put the overview tab on the far left but make folks land on the Compare tab for now. Are we not landing on the Overview tab right now because these arent complete or pretty yet (and the Compare tab is the shiny pretty feature we want to highlight at this point).  But ultimately I would expect us to land on the Overview tab.

**Comment by jmcmurry** [comment]
I have some ideas about the overview page composition and layout; there are
also some related tickets floating about.
Perhaps we can discuss together after the September release.

**Comment by kshefchek** [comment]
I've fixed locally, but holding off until I can test as SciGraph is having issues.
